src/lib/components/Creatibutors/CreatibutorsIdentifiers.js

```diff
@@ -0,0 +1,90 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 CERN.
+// Copyright (C) 2021 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { SelectField } from 'react-invenio-forms';
+import _unickBy from 'lodash/unionBy';
+
+export class CreatibutorsIdentifiers extends Component {
+  static propTypes = {
```

Comment:
:warning: This doesn't seem to be supported in Safari https://caniuse.com/mdn-javascript_classes_static_class_fields . Can they be assigned like we usually do?

Reply:
I tested it and it works...maybe babel or react traspiles it correctly out of the box as it is documented on their side!

Reply:
ah good!